packages/okta-auth-js/test/spec/mfa-enroll-activate.js

```diff
@@ -1,6 +1,14 @@
 jest.mock('cross-fetch');
 
-var util = require('../util/util');
+var util = require('@okta/test.support/util');
+var Q = require('q');
+
+function mockQDelay() {
```

Comment:
ditto. why write another mock method instead of using `util.mockQDelay`

Reply:
In this case we are mocking a library that the test/lib depends on (Q), I do not want to add Q as a dependency of test-support, we are phasing it out.